Sources/ApodiniDeployRuntimeSupport/ExportStructureCommand.swift

```diff
@@ -0,0 +1,84 @@
+//
+// This source file is part of the Apodini open source project
+//
+// SPDX-FileCopyrightText: 2019-2021 Paul Schmiedmayer and the Apodini project authors (see CONTRIBUTORS.md) <paul.schmiedmayer@tum.de>
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import ArgumentParser
+import Apodini
+import Runtime
+
+/// A CLI command responsible for exporting the structure of the web service.
+/// The export can be configured by providing options and arguments.
+/// The default implementation of this command exports the structure without any constraints.
+/// A deployment provider may define a subcommand to this to account for custom options.
+/// **Note**: Should not be called by the user directly
+public struct ExportStructureCommand: ParsableCommand {
+    /// Often used options that can be imported into a subcommand.
+    public struct ExportOptions: ParsableArguments {
+        @Argument(help: "The location of the json file")
+        public var filePath: String = "service-structure.json"
+        
+        @Option(help: "The identifier of the deployment provider")
+        public var identifier: String = "de.tum.in.ase.apodini.deploy"
+        
+        public init() {}
+    }
+    
+    public static var configuration = CommandConfiguration(
+        commandName: "export-ws-structure",
+        abstract: "Export web service structure",
+        discussion: "Exports an Apodini web service structure",
+        version: "0.3.0"
+    )
+    
+    public func run() throws {
+        ExportStructureCommand.helpMessage(columns: nil)
+        ExportStructureCommand.exit(withError:
+                                    ApodiniDeployRuntimeSupportError(message: "Calling this command directly is not supported.")
```

Comment:
This intention looks a bit strange, I would indent it +4 spaces from the previous line.